app/models/orm/migrations/env.py

```diff
@@ -98,7 +98,7 @@ def run_migrations_online():
         with context.begin_transaction() as transaction:
             context.run_migrations()
             if "dry-run" in context.get_x_argument():
-                print("Dry-run succeeded; now rolling back transaction")
+                print("Dry-run succeeded; now rolling back transactionapp")
```

Comment:
Was this intentional?  I'm assuming it should still just be "now rolling back transaction", but I could certainly be missing something!

Reply:
Not intentional, but included in a commit cherry-picked from another PR. I didn't bother to fix it as I will be reverting the commit before merge anyway.